dockerfiles/hpcc/values.schema.json

```diff
@@ -144,6 +144,10 @@
           "type": "array",
           "items": { "type": "string" }
         },
+        "useChildProcesses": {
+          "type": "boolean",
+          "description": "Launch each workunit compile in its own container, or as a child process"
```

Comment:
sounds a little confusing? 
Perhaps "If true, compile in child process if true. If false (default) compile in separate container"